react/src/components/molecules/FooterLinks/index.js

```diff
@@ -4,19 +4,19 @@ import './style.css';
 import { SidebarHeading } from '../../../index';
 
 const FooterLinks = (footerLinks) => (
-  <section className="ma__footer-links">
+  <nav className="ma__footer-links" aria-label="footer links">
     {
       footerLinks.items.map((footerLinksNav, i) => (
         /* eslint-disable-next-line react/no-array-index-key */
         <FooterLinksNav {...footerLinksNav} showNavHeading={footerLinks.showNavHeading} key={`footerLinksNav_${i}`} index={i} />
       ))
     }
-  </section>
+  </nav>
 );
 
 
 const FooterLinksNav = (footerLinksNav) => (
-  <nav aria-labelledby={footerLinksNav.id} className={`ma__footer-links__items${footerLinksNav.showNavHeading ? ' ma__footer-links__items--heading' : ''}`}>
+  <div className={`ma__footer-links__items${footerLinksNav.showNavHeading ? ' ma__footer-links__items--heading' : ''}`}>
     {
       footerLinksNav.showNavHeading ? <SidebarHeading title={footerLinksNav.heading} level={2} /> : <h2 className="visually-hidden" id={footerLinksNav.id}>{footerLinksNav.heading}</h2>
```

Comment:
Please use `<h3>` instead of `<h2>` since we adds `<h2>` for the `section.ma__footer__nav`.
The heading doesn't need `id` since no need for the association with the removed `<nav>` anymore with `aria-labelledby`.  Please remove the `id`.